docsource/functionalRequirements.rst

```diff
@@ -1,2 +1,24 @@
 Functional Requirements
-========================
\ No newline at end of file
+========================
+
+1. Support the indexing of many data types (Genome, Assembly, Narrative etc). It is expected that there will be about 1000 types
+
+2. Support the indexing of versions of object types. It is expected that each object can have at most about 100 versions.
```

Comment:
I *think* this is talking about type versions right, not object versions? In which case object in the second sentence should be 'type' or 'object type'